src/JsPsych/trials/examples/camera/CameraStart.js

```diff
@@ -0,0 +1,124 @@
+import htmlButtonResponse from '@jspsych/plugin-html-button-response';
+
+// TODO: Move markup to JsPsych?
+import { photodiodeGhostBox } from '../../../../lib/markup/photodiode';
+import { baseStimulus } from '../../../../lib/markup/stimuli';
+
+import { language } from '../../../language';
+import { TASK_NAME } from '../../../constants';
+
+// TODO: This is a task, how do I pass which config file to use?
+// Hard code for now
+import config from '../../../config/home.json';
+import { initJsPsych } from 'jspsych';
+
+// Get the ipcRender if running in an  electron window
+// TODO 192: Is it okay for this to start undefined?
+// TODO 192: Add warning to trial if not running in electron?
+let ipcRenderer = false;
+// TODO: Camera can now be accessed by the browser
+// if (config.USE_ELECTRON) {
+// if (config.equipment.camera === true) {
+//   ipcRenderer = window.require('electron').ipcRenderer;
+// }
+
+// ! This will break, need only a single JsPsych instance for the whole app
+const JSPSYCH = initJsPsych();

```

Comment:
Is there any reason this couldn't be passed into `createCameraStartTrial`? Or, if it can't, would it be worth exporting a standalone JSPSYCH object from a different file so it can be important globally?

Reply:
With JsPsych 7 you now have  a single, "central", JsPsych instance like you're talking about. Some of these trials were written  before that which is why they're spinning up their own instances 😞 

Moreover, we initialize our JsPsych instance inside the `<JsPsychExperinment />` which is _after_ the end user will create their trials. It's definitely an issue we'll have to come up with a more elegant solution for

Reply:
Ugh, don't like that... Feel free to resolve this then. If you want to brainstorm a better way to set that up, I'd be happy to help, but that's probably outside the scope of this issue (and I'm sure you'll figure out something good anyway).